cli/report/report.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package report
+
+import (
+	"context"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/open-policy-agent/opa/loader"
+	"github.com/open-policy-agent/opa/rego"
+)
+
+// GenerateReports takes CAI output and generate inventory report content based on report queries
+func GenerateReports(caiPath string, reportQueryPath string, reportOutputPath string, reportFormat string) error {
```

Comment:
nit. It'd be good to have clearer documentation on the expected inputs/outputs *or* don't export this function at all.

Reply:
Function needs to be exported because it's used by cmd.go

Updated function description and parameter names in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/315/commits/2c5e2414d3ca6cf6f707b03a5a007b491ad49daf